application.js

```diff
@@ -11,9 +11,6 @@ securableEndpoints = [];
 
 var app = express();
 
-// Enable CORS for all requests
-app.use(cors());
```

Comment:
Yes, if you add cors() middleware, the request can no longer be proxied. It waits indefinitely.

I'm not sure why's that, so I will leave it as open question for now.